Competition2018/src/main/java/competition/subsystems/elevator/commands/CalibrateElevatorTicksPerInchCommand.java

```diff
@@ -20,22 +20,16 @@
     OperatorInterface oi;
 
     /**
-     * This command needs to require all possible subsystems - while calibrating the elevator we DO NOT WANT
-     * ANYTHING ELSE TO MOVE.
+     * This command needs to require all possible subsystems - while calibrating the elevator we DO NOT WANT ANYTHING
+     * ELSE TO MOVE.
      */
     @Inject
-    public CalibrateElevatorTicksPerInchCommand(
-            ElevatorSubsystem elevator,
-            DriveSubsystem drive,
-            GripperDeploySubsystem wrist,
-            GripperIntakeSubsystem intake,
-            LeanSubsystem leaner,
-            ClimbSubsystem climber,
-            ClimberDeploySubsystem climbDeploy,
-            OperatorInterface oi) {
+    public CalibrateElevatorTicksPerInchCommand(ElevatorSubsystem elevator, DriveSubsystem drive,
```

Comment:
Hmph...

1. Why do we want to prevent everything else?
2. Why do we believe future programmers will add their subsystems to this list?

Reply:
I don't know, I did not write this

Reply:
1) The idea is that since this repurposes a joystick that may be used in other circumstances, the only safe thing to do is control all subsystems so that it doesn't accidentally actuate another subsystem.

2) In theory all robot actuation has been taken care of with the existing list, but there is a vulnerability here. Can you suggest a better way to deal with new actuation systems coming online?